bucket/runcat.json

```
@@ -1,13 +1,13 @@
 {
-    "version": "1.11",
+    "version": "2.0",
     "description": "A cute running cat animation on your windows taskbar (based on CPU load)",
     "homepage": "https://github.com/Kyome22/RunCat_for_windows",
     "license": "Apache-2.0",
-    "depends": "extras/windowsdesktop-runtime",
+    "depends": "extras/windowsdesktop-runtime-lts",
```

```suggestion
    "depends": "extras/windowsdesktop-runtime",
```
The package `windowsdesktop-runtime-lts` is for .NET 6.0 apps. However *RunCat* still only supports .NET 5.0.
https://github.com/Kyome22/RunCat_for_windows/issues/86

The package names for `windowsdesktop-runtime` look a bit confusing, but we will check dependent packages, and sort it out in a few days.